### ISSUE TITLE
Pencil crash (Bug)

### ISSUE THREAD

**Issue by marcello257** [pencil2d/pencil#168]
By Copying Timeline animation pictures there became a Bug problem so ‘Pencil’ stopped working and crashed again and again!!!
By the way...before the crash the image I was copying wasn’t seen anymore, Pencil did show only a gray canvas!

After restarting the programme (by wich I lost a few images!) Pencil copyied 4 or 5 images and then crashed again!!!

This is a very frustrating problem offcourse!

**Comment by CandyFace** [comment]
These kind of issues are hard to follow when there's no info as to which version is being used nor how to reproduce the said bug. Given the limited amount of developers and time constrains, it simply takes too much effort and time for us developers to decipher issues like this. 

i'm therefore closing this issue. If the problem still persists in the newest nightly build, feel free to make a new post about it.
